model/vcs/metrics.yaml

```diff
@@ -0,0 +1,139 @@
+groups:
+  - id: metric.vcs.repository.change.count
+    type: metric
+    metric_name: vcs.repository.change.count
+    brief: 'The number of changes (pull requests) in a repository, categorized by their state (either open or merged)'
+    instrument: updowncounter
+    unit: "{change}"
+    stability: experimental
+    attributes:
+      - ref: vcs.change.state
+        requirement_level: required
+      - ref: vcs.repository.url.full
+        requirement_level: required
+  - id: metric.vcs.repository.change.time
+    type: metric
+    metric_name: vcs.repository.change.time
```

Comment:
similar to my other comment on naming, please consider `vsc.change_request.duration`

Reply:
Renamed in 536a2a7defc49a81e598b228848258ff5201e94f.